### PR TITLE
feat: Add Reusable Feedback Popup

### LINKED ISSUE #7: User feedback popups
**Description of feature/enhancement**
Create multiple reusable feedback popups for the following cases
- error
- succes

if you have any other use case you can also make for those

### PR DESCRIPTION
**Pull request by CeereeC**
Closes #7 .

**Overview of changes:**

- Added the FeedbackPopup component 

**Anything you'd like to highlight/discuss:**
- This component was built using the MUI Alert component
- The prop "severity" is the same prop used to pass into the Alert. There are only 4 values possible: "success" , "info", "warning", "error"
- The FeedbackPopup can currently be seen in the Home page, but I will remove it before the PR is merged.

### CODE REVIEW

File: components/FeedbackPopup/index.tsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import { Alert, AlertColor, AlertTitle, IconButton, Zoom } from '@mui/material'
+import CloseIcon from '@mui/icons-material/Close'
+
+const FeedbackPopup = ({
+  open,
+  setOpen,
+  title,
+  desc,
+  severity,
```
[the remark this answers is not in the document]
Comment (wilinetan) [reply components/FeedbackPopup/index.tsx]: i think parent can be something like this
```
const ParentComponent = () => {
const [open, setOpen] = useState(false)
const { data, isValidating, error = true } = useSWR('/getData')

  return (
    <div>
      <FeedbackPopup
       // mainly just looking our for errors to cause this alert to show itself
        isOpen={error}
        title="Title"
        desc="Description of Information"
        severity="info"
       // this is a Alert prop that I decided to override
        variant="outlined"
      />
      <h1>Mercy Relief Internal Portal</h1>
      <Button onClick={() => setOpen(!open)}>Popup</Button>
    </div>
  )
}
```

File: components/FeedbackPopup/index.tsx [hunk 2]
```diff
@@ -1,41 +1,63 @@
...
             </IconButton>
           }
-          severity={severity}
-          sx={{ mb: 2, width: '50%', position: 'absolute', right: '25px' }}
+          severity={props.severity}
+          sx={{
+            minWidth: '250px',
+            ...props,
+          }}
         >
```
Comment (wilinetan) [inline components/FeedbackPopup/index.tsx:56]:
```
 <Alert
          action={
            <IconButton aria-label="close" size="small" onClick={handleClose}>
              <CloseIcon fontSize="inherit" />
            </IconButton>
          }
          sx={{minWidth: '250px'}}
         // severity prop is already given as a prop so we just need to add it in here. allows for user to override any other props
          {...props}
        >
```

Comment (CeereeC) [reply components/FeedbackPopup/index.tsx]: Alright I've added the change

**Review by wilinetan — APPROVED** [review]
LGTM. thanks! 👍 can merge after removing the fpb page

**Pull request merged**